### DISCUSSION TITLE
Examples in the doc comments?

### DISCUSSION THREAD

**Discussion by GoEddie** [dotnet/spark#645] [Ideas]
I was looking at the documentation page that is built from the comments on each method and wondered if there was a way we could add a small example to each method in C# and F# so they could be displayed on these pages https://docs.microsoft.com/en-us/dotnet/api/microsoft.spark.sql.dataframereader.schema?view=spark-dotnet

I know there is an <example> and <code> tag - can / should we use these as we add in new methods?

ed

**Comment by GoEddie** [comment]
Just another thought after reading that link specifically (https://docs.microsoft.com/en-us/dotnet/api/microsoft.spark.sql.dataframereader.schema?view=spark-dotnet), the remarks look a bit strange because of the line breaks - maybe worth considering removing the 100 char line limit for comments.

**Comment by Niharikadutta** [comment]
This is a good point, we should take a look to make the API docs more readable.
